lib/ansible/module_utils/docker_common.py

```diff
@@ -110,6 +110,8 @@
 
 
 if not HAS_DOCKER_PY:
+    docker_version = None
```

Comment:
Is there a reason you add this here? Is this used by any of the modules?

Reply:
Ah, and that doesn't work if `docker_version` has not been defined?

Reply:
It doesn't work if `docker_version` has not been defined.